mathics/doc/latex/doc2latex.py

```diff
@@ -89,6 +89,7 @@ def try_cmd(cmd_list: tuple, stdout_or_stderr: str) -> str:
         ["GhostscriptVersion", ("gs", "--version"), "stdout"],
     ):
         versions[name] = try_cmd(cmd, field)
+    versions.update(version_info)
```

Comment:
consider including the matplotlib version too, if available.

Reply:
Good idea. Done in d5c6ea13